docker/compute_worker/compute_worker.py

```diff
@@ -127,6 +128,14 @@ def delete_files_in_folder(folder):
             shutil.rmtree(file_path)
 
 
+class ExecutionTimeLimitExceeded(Exception):
+    pass
+
+
+def alarm_handler(signum, frame):
```

Comment:
I assume the alarm stuff doesn't take a lambda for some reason? Or did python get rid of inline lambda, can't recall now..

Reply:
tried to do a lambda, it didn't like it. At all...